types/ckeditor__ckeditor5-core/src/command.d.ts

```diff
@@ -17,7 +17,7 @@ import { Emitter, EmitterMixinDelegateChain } from '@ckeditor/ckeditor5-utils/sr
  * By default, commands are disabled when the editor is in {@link module:core/editor/editor~Editor#isReadOnly read-only} mode
  * but commands with the {@link module:core/command~Command#affectsData `affectsData`} flag set to `false` will not be disabled.
  */
-export default class Command implements Observable {
+export default abstract class Command implements Observable {
```

Comment:
Is this more in line with the underlying library?  It seems like it could break people and I'd only want to do that to fix a bug.

Reply:
I would say yes. Much like `Plugin`, consumers are expected to extend `Command` since the base `execute()` does nothing.
https://github.com/ckeditor/ckeditor5/blob/v32.0.0/packages/ckeditor5-core/src/command.js#L232

Reply:
IMO it's just no-op, I expect opposite should be mark explicit in some way or another:
https://jsdoc.app/tags-abstract.html
(that's the same dillema here like reviewing this `dispose` method

Reply:
I thought I've already marked ckeditor's `Plugin` as abstract too, but since it's not the case, I will revert this change to `Command`.

Reply:
@fedemp you own the implementation. you're probably the best here to pick proper implementation for CD editor types users. We are just pointing those minor problems or 'unknowns' details, that are to be explained during PR. We will pick what you stick with

Reply:
I rather keep them as non abstract. Consumers should check how `Command` works, and know that they have to extend it.